content/docs/tour/basics-destroying.md

```diff
@@ -1,73 +1,3 @@
 ---
-title: Destroying
-menu:
-  tour:
-    parent: tour
-    weight: 7
+redirect_to: https://www.pulumi.com/docs/reference/cli/pulumi_destroy/
```

Comment:
Nit: We just need `/docs/reference`, right? That would also mean the redirect would work when running locally, etc.

Reply:
This should be a relative link. I missed cleaning this up. Will fix.